scripts/arangodb-docker-container-setup/docker-container-scripts/js/setup.js

```diff
@@ -52,4 +53,26 @@ collectionNames.forEach(collectionName => {
   else {
     print(`Collection "${collectionName}" could not be created`);
   }
-});
+}
+
+print('Edge Collections')
+const edgeCollectionName = 'resource_connections';
+print(`Attempting to add collection "${edgeCollectionName}" to ${process.env.ARANGO_DB_NAME}`);
+// TODO: need to pass in collection type document or edge
+if (db._createEdgeCollection(edgeCollectionName)) {
+  print(`Created Edge Collection ${edgeCollectionName}`);
+  if (testDataFlag) {
+     testData.edges.forEach(edge => {
+        print(`Attempting to add data to collection "${edgeCollectionName}"`);
+        if (db._collection(edgeCollectionName).save(edge)) {
```

Comment:
The ArangoSH API is super funky. Are you saying that this is syncrhonous and it blocks the thread until it's done? That kind of sucks actually. But it looks like it is so [here](https://www.arangodb.com/docs/stable/data-modeling-documents-document-methods.html#insert--save).

Reply:
Not sure what the best approach is here.  Is this an issue if it's just a one-time load rather than part of the rest of the functionality of the app?